src/core/Akka.Streams.Tests.TCK/Akka.Streams.Tests.TCK.csproj

```diff
@@ -1,4 +1,5 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
+<Import Project="..\..\common.props" />
```

Comment:
That part exists in the othe proj files.
There was a warning about it in Visual Studio.